.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+indent_style = tab
+indent_size = 4
+
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Oh I wanted to bring this up for a while, thanks for adding it. Honestly, I think we should enable this for all of our IDEs for all of our projects...

Reply:
Maybe we can bring it up on a backlog grooming...